content/_releases/22/34.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Product Update - v22.34
+description: Details of the product update v22.34 from 25th of August 2022.
+layout: rel
+version: "22.34"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-08-25
+redirect_from:
+  - /releases/22.34.html
+---
+
+## Improvements and Updates
+
+### Meaningful error for invalid SSH key entry
```

Comment:
Informative error messaging for invalid SSH key entries

Reply:
Done